nexus/proposals/models.py

```diff
@@ -0,0 +1,49 @@
+# Third Party Stuff
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from extended_choices import Choices
+
+# Nexus Stuff
+from nexus.base.models import TimeStampedUUIDModel
+
+
+class Proposal(TimeStampedUUIDModel):
+    PROPOSAL_KIND = Choices(
+        ('TALK', 'talk', _('Talk')),
+        ('DEV_SPRINT', 'dev_sprint', _('Dev Sprint')),
+        ('WORKSHOP', 'workshop', _('Workshop'))
+    )
+
+    LEVELS_CHOICES = Choices(
+        ('BEGINNER', 'beginner', _('Beginner')),
+        ('INTERMEDIATE', 'intermediate', _('Intermediate')),
+        ('ADVANCED', 'advanced', _('Advanced'))
+    )
+
+    STATUS_CHOICES = Choices(
+        ('RETRACTED', 'retracted', _('Retracted')),
+        ('ACCEPTED', 'accepted', _('Accepted')),
+        ('UNACCEPTED', 'unaccepted', _('Unaccpted')),
+        ('SUBMITTED', 'submitted', _('Submitted'))
+    )
+
+    title = models.CharField(_('Title'), max_length=120, null=False, blank=False)
+    speaker = models.ForeignKey('users.User', on_delete=models.CASCADE, verbose_name=_('Speaker'),
```

Comment:
On a second thought, I think this should be `author` of the proposal and speakers can be just names. But let's think about this first. (No need to change)